README.md

```diff
@@ -779,7 +779,7 @@ For instance this will work:
 let schema = number().test(
   'is-42',
   "this isn't the number i want",
-  (value) => value != 42,
+  (value) => value === 42,
```

Comment:
```suggestion
  (value) => value == null || value === 42,
```

Reply:
Are you saying you want the test to pass if the value is null?

Reply:
if its undefined or null. that is handled by `required`, `defined` etc